migrations/1597864779562_create-items.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+import { Item } from '../src/Item'
+
+const tableName = Item.tableName
+
+export const up = (pgm: MigrationBuilder) => {
+  pgm.createTable(
+    tableName,
+    {
+      id: { type: 'UUID', primaryKey: true, unique: true, notNull: true },
+      name: { type: 'TEXT', notNull: true },
+      description: { type: 'TEXT' },
+      eth_address: { type: 'TEXT', notNull: true },
```

Comment:
Maybe `eth_address` can be interpreted to the Ethereum address of the item, What do you think of using `owner` or `creator`? We know that in the future we will need to have off/on-chain permissions, so we need to use a name that won't hurt.

Reply:
The problem is that AssetPacks, Assets, etc use `eth_address` to represent the user's address. I thought of owner as well when we where setting up the database but juanca reminded us of that. Furthermore, parts of the code rely on that property being called `eth_address` to work generically. Like Ownable